app/pages/Covid/Covid.jsx

```diff
@@ -15,30 +16,19 @@ export default class Covid extends React.Component {
         <article className={styles.covidPage} id="covid">
           <header className={styles.covidHeader}>
             <h1>
-              SF Service Guide
-              <br />
-              COVID-19 resource guide
+            Coronavirus resources for Individuals-In-Need            
             </h1>
             <p>
-              This is living document from
-              {' '}
-              <a href="ShelterTech.org">ShelterTech</a>
-              {' '}
-              of resources for individuals experiencing homelessness. We will be updating the list continuously, when services change, to ensure you have the most up to date information.
-              Please email
-              {' '}
-              <a href="mailto:contact@sheltertech.org">contact@sheltertech.org</a>
-              {' '}
-              to suggest edits.
-              See what types of places are open or closed at
-              {' '}
-              <a href="sf.gov/stay-home-except-essential-needs">sf.gov/stay-home-except-essential-needs</a>
+              This list is compiled to link individuals experiencing homelessness to resources.  We will be updating the list continuously, when services change. Please email <a href="contact@sheltertech.org">contact@sheltertech.org</a> to suggest edits.
```

Comment:
Might be worth throwing a `mailto:` on here so this is recognized as an email address. ie `href="mailto:contact@sheltertech.org"`

Reply:
good idea, i'll add that